GoMeet/src/main/java/com/google/sps/data/Location.java

```diff
@@ -63,4 +63,23 @@ public int getVoteCount() {
   public String getKeyString() {
     return keyString;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Not entirely sure what this function does - would be helpful to add a comment describing the functionality and how/when it would be used